ime/app/src/main/res/values-is/strings.xml

```diff
@@ -0,0 +1,958 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This entire file will be overwritten from Crowdin: If you wish to help translate to Icelandic, please head to https://ask_translate.evendanan.net/project/anysoftkeyboard/invite

Reply:
I'd put this into another PR. What do we do here ? Remove the file ? Btw: Firefox complains about the above URL

Reply:
Yes, remove the file.
If you wish to donate Icelandic translation, use Crowdin: https://crwd.in/anysoftkeyboard (I've changed the URL, as you mentioned, there is a issue with the cert)

Reply:
File removed. Thx for the hint on how the I8N works for this project.